packages/mdc-textfield/README.md

```diff
@@ -142,32 +161,37 @@ See [here](../mdc-notched-outline/) for more information on using the outline su
 
 > _NOTE_: Do not use `mdc-line-ripple` inside of `mdc-text-field` _if you plan on using `mdc-text-field--outlined`_. Bottom line should not be included as part of the DOM structure of an outlined text field.
 
-#### Helper Text
+### States
```

Comment:
Not entirely sure what the thought was here, but I think we just have disabled as a direct subheading under variants in other components?

Reply:
Gotcha...thought it was more of a `state` than `variant`, but I'll change it to be more consistent.